core/git_mixins/branches.py

```diff
@@ -30,7 +30,7 @@ def _parse_branch_line(line):
         if not line:
             return None
 
-        pattern = r"(\* )?(remotes/)?([a-zA-Z0-9\-\_\/\.]+(?<!\.lock)(?<!\/)(?<!\.)\b) +([0-9a-f]{40}) (\[([a-zA-Z0-9\-\_\/\\]+)(: ([^\]]+))?\] )?(.*)"
+        pattern = r"(\* )?(remotes/)?([a-zA-Z0-9\-\_\/\.]+(?<!\.lock)(?<!\/)(?<!\.)\b) +([0-9a-f]{40}) (\[([a-zA-Z0-9\-\_\/\.\\]+)(: ([^\]]+))?\] )?(.*)"
```

Comment:
@divmain: I think the `\\` at the end of the remote matching part of the pattern (`([a-zA-Z0-9\-\_\/\.\\]+)`) might be a typo, as far as I know there shouldn't be any `\`s here? If it's a typo I can update the commit to remove it, just wanted to check with you first.